src/base/static/components/ui-elements/editor-button.scss

```diff
@@ -46,14 +46,17 @@
 
 .editor-button__icon {
   color: #fff;
-  margin-right: 7px;
   float: left !important;
 
   &:before {
     font-family: "FontAwesome";
   }
 }
 
+.editor-button__icon--with-right-margin {
```

Comment:
`--with-right-margin` feels too much like you're just restating the contents of this CSS statement, instead of making a semantic statement about when this is necessary. Sounds like putting the margin on the label itself may be a better solution than this class even existing, however.